### ISSUE TITLE
UAT keyword

### ISSUE THREAD

**Issue by gilleslandais** [augustfly/DOI4Archives#13]
That's a detail

I propose to encourage dataset provider to use UAT keywords

Lot's  of advantages using UAT: which are also encouraged in the registry. It is based on a semantic (UAT thesaurus)

Datacite propose to specify the schema URI whoich can be completed with a classificationCode.

eg: 
```
<subject schemeURI="https://astrothesaurus.org/uat/" subjectScheme="UAT" classificationCode="1054">Milky Way Galaxy</subject>
```
Question:  
- granularity level: to put or not  the ```classificationCode``` ? (is the UAT url persistent ? can we use ```classificationCode``` ?)

**Comment by gilleslandais** [comment]
After discussion with  @BaptisteCecconi, it sounds preferable to priviledge valueURI than schemeURI+classificationCode.

The most often a RDF term has his own URL  - classificationCode is an alternative when RDF terms are defined in a same document.(furthermore schemeURI exists already in datacite 4.0)

eg: 
```<subject valueURI="https://astrothesaurus.org/uat/1054" subjectScheme="UAT">Milky Way Galaxy</subject>```

**Comment by augustfly** [comment]
I agree with encouraging UAT keywords. I'm not clear on how to write that advisory when the UAT coverage is variable, esp in helio and planetary @aaccomazzi

**Comment by gilleslandais** [comment]
a detail: there is here a recommendation a little different in VO registry which proposed to use IVOA-UAT(https://www.ivoa.net/documents/uat-as-upstream/).  (SKOS semantics is used to link IVOA to UAT)

however, I don't know if the IVOA semantic added more planetary concept..

**Comment by acraugh** [comment]
Value URIs are intended to be programmatically readable - there is no requirement that they be human-readable, and often they are not (because numbers can be more efficient to deal with than random strings). With respect to the DataCite "classificationCode", this attribute was added as an alternative to "valueURI" for vocabularies that have codes defined in documentation, but do not have APIs to databases that would allow those codes to be programmatically recognized and compared - and thus are not "value URIs" in a practical sense.
